test/sanity/validate-modules/module_args.py

```diff
@@ -75,7 +77,26 @@ def setup_env(filename):
                 del sys.modules[k]
 
 
-def get_argument_spec(filename):
+def get_ps_argument_spec(filename):
+    # This uses a very small skeleton of Ansible.Basic.AnsibleModule to return the argspec defined by the module. This
+    # is pretty rudimentary and will probably require something better going forward.
+    cmd = ['test/sanity/validate-modules/ps_argspec.ps1', filename]
+
+    proc = subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE, shell=False)
```

Comment:
I'd check for `pwsh` and give a more user-friendly error message if it's not available.

When I run this without `pwsh` available I get an error like:

```
ERROR: lib/ansible/modules/windows/win_ping.ps1:0:0: E321 Exception attempting to import module for argument_spec introspection, 'env: pwsh: No such file or directory
'
```